hardhat.config.ts

```diff
@@ -12,6 +12,12 @@ module.exports = {
     compilers: [
       {
         version: '0.8.16',
+        settings: {
```

Comment:
Why is this neccessary?

Reply:
The hypERC20Votable contract size exceeds the maximum contract size. Thus we need an optimizer in order to deploy it.